server.js

```diff
@@ -0,0 +1,16 @@
+const express = require('express')
+const bodyParser = require('body-parser')
+const path = require('path')
+const PORT = process.env.PORT || 3000
+
+const app = express()
+
+const index = require('./routes/index')
```

Comment:
Refactor this as discussed: name the resource ('questions') and put the versioning inside the index.